### PR TITLE
adds idet filter for checking interlacement pattern status

### PR DESCRIPTION
**Pull request by ablwr**
As referenced and described in @dericed's http://www.tate.org.uk/research/publications/sustaining-consistent-video-presentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -634,6 +634,29 @@
...
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Check video file interlacement patterns</h3>
+          <p><code>ffmpeg -i <i>input file</i> -vf idet -f null -</code></p>
+          <p></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-vf idet</dt><dd>This calls the <a href="http://www.tate.org.uk/research/publications/sustaining-consistent-video-presentation">idet (detect video interlacing type) filter</a>.</dd>
```
Comment (retokromer) [inline index.html]: I guess in the other recipes we used `-filter:v`. Therefore we should here either do the same or at least indicate that `-vf` is an alias.

Comment (ablwr) [inline index.html]: You are right. I updated!

**Comment by retokromer** [comment]
LGTM & merged. Thank you!

File: index.html (hunk 1 above)
Comment (retokromer) [inline index.html]: I know, I am a pedantic, but it facilitates very much the scripting, when syntax is used consistently.

Comment (ablwr) [inline index.html]: No, I agree, especially for beginners it is good to see consistency in a verbose manner.

**Pull request merged**